frontend/src/app/page.tsx

```diff
@@ -1,12 +1,21 @@
-import MatchList from '@/components/MatchList'
+"use client"
+import LikeList from '@/components/LikeList'
 import CardScreen from '@/components/CardScreen'
+import {useState} from "react";
```

Comment:
nitpick (non-blocking): noticed some inconsistency with spacing. Since we're not running prettier automatically right now, it would be a good idea to run prettier before making a PR.